__tests__/app.test.js

```diff
@@ -217,4 +219,59 @@ describe("POST /api/articles/:article_id/comments",()=>{
             expect(response.body.msg).toBe('Bad Request');
         });
     })
+})
+describe("PATCH /api/articles/:article_id",()=>{
+    test("updates the article, responds with 200 status and the updated Article",()=>{
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes: -100})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject({
+                article_id: expect.any(Number),
```

Comment:
I would want to see this a little more specific - we want to test that a) votes are increased and b) they're increased on a specific article. So I would have liked to have seen the updated number etc